src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomMobile.java

```diff
@@ -0,0 +1,285 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.FandomAdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.HiViUap;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsFandomObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers.SoundMonitor;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.concurrent.TimeUnit;
+
+@InBrowser(
+  browser = Browser.CHROME,
+  emulator = Emulator.GOOGLE_NEXUS_5
+)
+@Test(
+  groups = {"AdsUapHiViFandomMobile"}
+)
+public class TestAdsUapHiViFandomMobile extends AdsFandomTestTemplate {
+
+  private static final double IMPACT_STATE_ASPECT_RATIO = 272.0 / 153.0;
+  private static final double RESOLVED_STATE_ASPECT_RATIO = 640.0 / 213;
+  private static final String TLB_SLOT_ID = "gpt-top-leaderboard";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "gpt-top-leaderboard" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)